src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -545,7 +545,11 @@ public async Task Initialize(HostWaitLoop waitLoop, CancellationToken token)
 
             try
             {
-                await this.MICommandFactory.EnableTargetAsyncOption();
+                Results res = await this.MICommandFactory.ListTargetFeatures();
+                if (res != null && res.Contains("async"))
```

Comment:
Sorry for the confusion - now that you are returning an empty results instead of `null`, you don't need the null checks anymore.